python/tests/test_strategies.py

```diff
@@ -41,6 +43,31 @@ def test_backtest():
                 strategy
             )
             assert cerebro is not None, errmsg
+
+            data_disclosures = get_stock_data(
+                "JFC", "2015-01-01", "2020-09-30", source="phisix"
+            )
+
+            sentiments_disclosures = get_disclosure_sentiment(
```

Comment:
Can we make the date range as short as possible? perhaps 1 month? Let's be mindful that this test will run every time a commit happens, which could be a lot of requests to the website.

I suspect we'll get cases where tests will fail due to the site being down.

Reply:
Actually, maybe better if we forego the pse edge data pull and just run the sentiment pipeline on a saved copy of historical disclosures (similar to the previous sentiment tests i believe).

Reply:
got it!